vcd/resource_vcd_catalog.go

```diff
@@ -217,6 +255,33 @@ func genericResourceVcdCatalogRead(d *schema.ResourceData, meta interface{}) err
 		return err
 	}
 
+	dSet(d, "catalog_version", adminCatalog.AdminCatalog.Catalog.VersionNumber)
+	// dSet(d, "owner_name", adminCatalog.AdminCatalog.Owner.User.Name)
+
+	vAppTemplates, err := catalog.QueryVappTemplateList()
```

Comment:
I think it's to heavy to call all templates and medias to know the numbers. Catalog record (from query) has these numbers:
```
<CatalogRecord name="cat-vb" description="vb catalog" isPublished="false" isShared="false" creationDate="2022-02-21T13:13:37.525Z" orgName="vb" ownerName="system" numberOfVAppTemplates="2"
```

Reply:
@vbauzysvmware please check now. I used your suggestion and now things are simpler and more understandable!